packages/server/package.json

```diff
@@ -38,7 +38,7 @@
     "axios-retry": "^3.5.1",
     "chai": "^4.3.6",
     "ethers": "^6.7.0",
-    "execution-apis": "git://github.com/ethereum/execution-apis.git#584905270d8ad665718058060267061ecfd79ca5",
+    "execution-apis": "git://github.com/ethereum/execution-apis.git#7907424db935b93c2fe6a3c0faab943adebe8557",
```

Comment:
just being curious, where did you get this updated number?

Reply:
From the execution apis repo